server/views/pages/bookedtoday/arrivals/searchforexisting/search/searchForExistingRecord.njk

```diff
@@ -7,10 +7,7 @@
             <div class="govuk-grid-column-two-thirds">
                 <span class="govuk-caption-xl">Confirm a prisoner's arrival</span>
                 <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
-                <p>We could not check if this person has an existing prisoner record because the Person Escort Record does not include a
-                    valid PNC or prison number.</p>
-                <p>Check their personal details are correct and add a PNC or prison number if you have one. This information will be
-                    used to search for an existing prisoner record.</p>
+                <p>Check this person's details are correct and add or change their PNC or prison number if you have one.</p>
```

Comment:
New generic content as this is used both when we have no match and no identifiers and also when we have automatically matched and the user wants to ignore it